src/Symfony/Bundle/FrameworkBundle/Tests/Functional/CachePoolsTest.php

```diff
@@ -50,7 +60,7 @@ public function testRedisCustomCachePools()
         try {
             $this->doTestCachePools(array('root_config' => 'redis_custom_config.yml', 'environment' => 'custom_redis_cache'), RedisAdapter::class);
         } catch (\PHPUnit_Framework_Error_Warning $e) {
-            if (0 !== strpos($e->getMessage(), 'unable to connect to 127.0.0.1')) {
+            if (0 !== strpos($e->getMessage(), 'unable to connect to')) {
```

Comment:
not php warning either generated by phpunit? then feel free to remove the copy/paste :)